src/yeahml/build/components/metric.py

```diff
@@ -1,62 +0,0 @@
-import inspect
```

Comment:
I think this file shouldn't be deleted (unless I'm missing something by comparing it to the wrong commit, but the latest master needs this and some other files I'll mark w/[mark for keep] in the comments)